ehcache-core/src/main/java/org/ehcache/core/EhcacheManager.java

```diff
@@ -306,8 +306,12 @@ private <K, V> Cache<K, V> createCache(String alias, CacheConfiguration<K, V> or
     return cache;
   }
 
+  @SuppressWarnings("try")
   <K, V> InternalCache<K, V> createNewEhcache(String alias, CacheConfiguration<K, V> config,
                                         Class<K> keyType, Class<V> valueType) {
+
+    try(EhcachePrefixLoggerFactory.Context ignored = EhcachePrefixLoggerFactory.withContext("cache-alias", alias)){
```

Comment:
This blocks contents should be indented. I'm also pondering if inverting this to:
```java
EhcachePrefixLoggerFactory.withContext("cache-alias", alias, () -> {
   ...
});
```
That would avoid the need to suppress the try-with-resources warning and also make it impossible to forget to close the context.